pilot/pkg/model/config_test.go

```diff
@@ -642,6 +701,21 @@ func TestAuthenticationPolicyConfig(t *testing.T) {
 				t.Errorf("AutheticationPolicy(%s:%d) => expected %#v but got %#v",
 					testCase.hostname, testCase.port, expected, out)
 			}
+
+			// Verify jwks_uri is got correctly through mock openID discovery server if not set in the auth policy.
+			if testCase.expected == "once" || testCase.expected == "twice" {
```

Comment:
"once" -> "mockonce" ?
Alternative, add a function to create "golden data", and set the expected value accordingly.
(btw, you won't need to change this test if move the getJwksUri into authentication.go as mentioned above)

Reply:
done